### DISCUSSION TITLE
A command to the check the luck stat of a player

### DISCUSSION THREAD

**Discussion by EdoanR** [Pryaxis/TShock#2075] [Ideas]
I don't know if this is possible, but could be really cool to see how much luck a player have, this is a really important mechanic. 
If someone is unfamiliar what i'm talking about, check out this [page](https://terraria.gamepedia.com/Luck) on wiki, pretty much explains everything

(I know that the Wizard give hints about it, but I want a more clear and fast way to know luck)

**Comment by hakusaro** [comment]
It's definitely possible. Arguably this is something that should be a plugin, but it could be a command. I believe it's possible for us to change luck too, so it's reasonable.

GitHub <notifications@github.com> wrote:
“

I don't know if this is possible, but could be really cool to see how much luck a player have, this is a really important mechanic.

If someone is unfamiliar what i'm talking about, check out this page on wiki, pretty much explains everything

(I know that the Wizard give hints about it, but I want a more clear and fast way to know luck)

**Reply by IraJacobs** [reply]
+1 to this request, both checking and modifying Luck.
